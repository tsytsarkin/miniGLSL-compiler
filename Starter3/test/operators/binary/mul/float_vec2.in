{ float a; vec2 b; c = a * b;}
