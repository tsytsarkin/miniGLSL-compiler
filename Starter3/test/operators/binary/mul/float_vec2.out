  1: { float a; vec2 b; c = a * b;}
 (SCOPE (DECLARATIONS (DECLARATION a float (DECLARATION b vec2))) (STATEMENTS (ASSIGN unknown c (BINARY vec2 * a b))))
